tests/DidCache.spec.ts

```diff
@@ -0,0 +1,126 @@
+import * as Base58 from 'bs58';
+import { Cas } from '../src/Cas';
+import { DidCache } from '../src/DidCache';
+import { DidDocument } from '@decentralized-identity/did-common-typescript';
+import { WriteOperation } from '../src/Operation'
+
+// Implementation of Did document update - no-op for testing purposes
+function didDocumentUpdate (didDoc: DidDocument, _operation: WriteOperation): DidDocument {
+  return didDoc;
+}
+
+const didDocJson = {
+  "@context": "https://w3id.org/did/v1",
+  "id": "did:sidetree:ignored",
+  "publicKey": [{
+    "id": "did:sidetree:didPortionIgnored#key-1",
+    "type": "RsaVerificationKey2018",
+    "owner": "did:sidetree:ignoredUnlessResolvable",
+    "publicKeyPem": "-----BEGIN PUBLIC KEY...END PUBLIC KEY-----\r\n"
+  }],
+  "service": [{
+    "type": "IdentityHub",
+    "publicKey": "did:sidetree:ignored#key-1",
+    "serviceEndpoint": {
+      "@context": "schema.identity.foundation/hub",
+      "@type": "UserServiceEndpoint",
+      "instances": ["did:bar:456", "did:zaz:789"]
+    }
+  }]
+};
+
+// Implementation of Did document create - return a dummy did document
+function didDocumentCreate (_operation: WriteOperation): DidDocument {
+  return new DidDocument(didDocJson);
+}
+
+
+// Implementation of a dummy cas class for testing - a simple hash map
+class DummyCas implements Cas {
```

Comment:
Please remove this class and move the implementation logic to MockCas so we don't have multiple test CAS classes.

Reply:
DummyCas actually is more general than MockCas. I simply moved the DummyCas to MockCas and removed the old MockCas.